src/django_upgrade/fixers/request_headers.py

```diff
@@ -58,6 +58,44 @@ def visit_Call(
         )
 
 
+@fixer.register(ast.In)
+def visit_In(
+    state: State,
+    node: ast.In,
+    parents: list[ast.AST],
+) -> Iterable[tuple[Offset, TokenFunc]]:
+    if (
+        type(parents[-1]) is ast.Compare
+        and len(parents[-1].comparators) == 1
+        and is_request_or_self_request_meta(parents[-1].comparators[0])
+        and type(parents[-1].left) is ast.Constant
+        and (raw_header_name := get_http_header_name(parents[-1].left.value))
+        is not None
```

Comment:
use parentheses to avoid black aligning the `is` and `and`
```suggestion
        and (
            (raw_header_name := get_http_header_name(parents[-1].left.value))
            is not None
        )
```